components/lvgl_esp32_drivers/lvgl_tft/rm67162.c

```diff
@@ -72,9 +72,9 @@ void rm67162_init(void)
 
 	//Reset the display
 	gpio_set_level(RM67162_RST, 0);
-	vTaskDelay(300 / portTICK_RATE_MS);//default 100ms
+	vTaskDelay(300 / portTICK_PERIOD_MS);//default 100ms
```

Comment:
you should not change any part of the code out of your task scope

Reply:
because my project did not compile !